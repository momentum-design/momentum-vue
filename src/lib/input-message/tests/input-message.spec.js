import { mount } from '@vue/test-utils';
import InputMessage from '../index.vue';

describe('InputMessage', () => {
  it('should match snapshot', () => {
    const wrapper = mount(InputMessage, {
      propsData: {
        message: 'test'
      }
    });
    expect(wrapper.html()).toMatchSnapshot();
  });

  it('should match render message with rendered text', () => {
    const wrapper = mount(InputMessage, {
      propsData: {
        message: 'test'
      }
    });

    expect(wrapper.find('.md-input__message').text()).toEqual('test');
  });
});

