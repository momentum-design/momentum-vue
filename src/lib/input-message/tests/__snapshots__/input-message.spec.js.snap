// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputMessage should match snapshot 1`] = `
<div role="alert" class="md-input__message">
  test
</div>
`;
